sw/airborne/modules/computer_vision/lib/vision/image.c

```diff
@@ -578,27 +608,24 @@ void image_show_points(struct image_t *img, struct point_t *points, uint16_t poi
  */
 void image_show_flow(struct image_t *img, struct flow_t *vectors, uint16_t points_cnt, uint8_t subpixel_factor)
 {
+  uint8_t color[4] = {255,255,255,255};
+  int size_crosshair = 5;
```

Comment:
In the future, things like this can be declared static so that they are not constantly redeclared. alternatively you can consider using the const descriptor, helps with code clarity.

Reply:
I made it static now

Reply:
This was not specific to this line but in general, often you define a variable with a constant value, in that case a static or const (or both) descriptor is interesting. Consider it in the future.